### DISCUSSION TITLE
Could it not be nice to have the option to select "Year" ?

### DISCUSSION THREAD

**Discussion by Daniel-PT** [giejay/Immich-Android-TV#30] [Ideas]
Hi 
I miss a feature to select "Year" in the app.
Sometimes we like to see picturs from our kids in year 2015. 
Hope it makes sense :)
Thanks!

**Comment by Daniel-PT** [comment]
@giejay is this meaby possible? 🙏

**Comment by giejay** [comment]
From an API perspective, it should be possible: https://immich.app/docs/api/search-metadata

But for now, Im working on different projects so I will not be able to pick this up.

**Comment by Daniel-PT** [comment]
Hi gj
I tryed to install the Nice immich app from you again.

But I still get the api error?

Have you found the solution?

Thanks!

søn. 23. jun. 2024 08.44 skrev GJ Compagner ***@***.***>:
